dirty_cat/datasets/_fetching.py

```diff
@@ -217,17 +219,16 @@ def _fetch_world_bank_data(
               saved as a CSV file.
 
     """
-    # See if file available locally :
     path = f"{data_directory}/{indicator_id}.csv"
-    csv_path = Path(path)
+    csv_path = Path(path).resolve()
```

Comment:
Here we should identify if the file is present locally.

Reply:
`resolve` only converts relative paths to absolute paths, without the file necessarily existing.
We check if the file actually exists line 225.

Reply:
Sorry, the question got lost in the comment below.